bve-build/src/main.rs

```diff
@@ -201,14 +201,13 @@ fn mangle_shader_name(combination: &ShaderCombination<'_>) -> String {
 }
 
 fn build_shaders() {
-    if Path::new("bve-render/shaders/spirv/.timestamp").exists()
-        && out_of_date("bve-render/shaders/compile", "bve-render/shaders/spirv/.timestamp")
+    if Path::new("bve-render/shaders/spirv").exists()
+        && out_of_date("bve-render/shaders/compile", "bve-render/shaders/spirv/")
     {
         remove_dir_all("bve-render/shaders/spirv").expect("Could not remove directory");
         println!("Removing out of date spirv directory")
     }
     create_dir_all("bve-render/shaders/spirv").expect("Could not create spirv directory");
-    File::create("bve-render/shaders/spirv/.timestamp").expect("Could not create timestamp");
```

Comment:
Why no more timestamp file?

Reply:
Forced a rebuild every time as a file changed in the tree. I can just check the directory timestamp.